openmanage/src/main/java/com/vmware/flowgate/openmanage/job/OpenManageJobService.java

```diff
@@ -135,18 +197,70 @@ private void syncMetricsDataJob(FacilitySoftwareConfig integration) {
 
    }
 
+   public List<Asset> handleServerAssets(DevicesResult<Server> serversResult, Map<Long,Asset> assetNumberMap ){
+      List<Server> servers = serversResult.getValue();
+      List<Asset> serverAssetsToSave = new ArrayList<Asset>();
+      for(Server server:servers) {
+         //filter by device status
+         if(DeviceStatus.NORMAL.getValue() != server.getStatus()) {
```

Comment:
status. we should save the data as far as the server still there.

Reply:
done